ode/src/KokkosODE_Types.hpp

```diff
@@ -51,6 +51,21 @@ struct ODE_params {
         min_step_size(min_step_size_) {}
 };
 
+enum newton_solver_status { NLS_SUCCESS = 0, MAX_ITER = 1, LIN_SOLVE_FAIL = 2 };
+
+struct Newton_params {
+  int max_iters;
+  double abs_tol, rel_tol;
+
+  // Constructor that only specify the desired number of steps.
+  // In this case no adaptivity is provided, the time step will
+  // be constant such that dt = (tend - tstart) / num_steps;
```

Comment:
Copy-pasted comment?

Reply:
Definitely, good catch!
I'll wait for the CI to complete, if it returns clean and might do a separate PR with no CI to change the text, no need to spend more compute resources on this.